app/google-services.json

```diff
@@ -0,0 +1,39 @@
+{
```

Comment:
Uhm file should be in gitignore 😳

Reply:
Cool okay, will do that. But I think this file doesn't have any confidential server APIs.